src/components/ui/dropzone.svelte

```diff
@@ -0,0 +1,71 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  export let classname = '';
+  export let inputclass = '';
+  export let id = '';
+  export let value = [];
+  export let accept = 'image';
+  export let maxSize = null;
+  export let multiple = true;
+  export let disabled = false;
+  export let disabledElements = [];
+  export let disabledQuerySelector = null;
+
+  $: files = value;
+
+  let filePlatform = null;
+  const dispatch = createEventDispatcher();
+  const fileClick = (e) => {
+    if (disabled) return;
+    if (disabledElements.length && disabledElements.some(el => {
+      return el.contains(e.target);
```

Comment:
Useless body. `param => { return f(param); }` is equivalent to `param => f(param)`.
Just write `.some(el => el.contains(e.target))`